bin/moodle-docker-compose

```diff
@@ -25,7 +25,14 @@ basedir="$( cd -P "$( dirname "$SOURCE" )/../" && pwd )"
 export ASSETDIR="${basedir}/assets"
 
 
-dockercompose="docker-compose -f ${basedir}/base.yml"
+docker compose version 2>&1 >/dev/null
```

Comment:
This does not work as intended, because of line 2: `set -e`. For `help set`:

> -e  Exit immediately if a command exits with a non-zero status.

Reply:
Thanks @dahrens, very good point. I made necessary changes.

To validate new approach I used small script which I tested on the machine with v2 `docker compose` and without:
```
#!/bin/bash
set -e

ver=$(docker compose version > /dev/null 2>&1 && docker compose version --short) || true

if [[ $ver =~ ^v2 ]]; then
  dockercompose="docker compose"
else
  echo 'Compose v2 is not available in Docker CLI, falling back to use docker-compose script'
  dockercompose="docker-compose"
fi

echo $dockercompose
```

This gave output matching the presence of docker compose v2.